test/unit/mongo/MongoTripodTestBase.php

```diff
@@ -309,13 +344,30 @@ protected function assertDoesNotHaveResourceTriple(ExtendedGraph $graph, $s, $p,
 
     protected function lockDocument($subject, $transaction_id)
     {
-        $tripod = new MongoTripod(LOCKS_COLLECTION, 'testing');
+        $collection = MongoTripodConfig::getInstance()->getCollectionForLocks('tripod_php_testing');
         $labeller = new MongoTripodLabeller();
         $doc = array(
             '_id' => array(_ID_RESOURCE => $labeller->uri_to_alias($subject), _ID_CONTEXT => MongoTripodConfig::getInstance()->getDefaultContextAlias()),
             _LOCKED_FOR_TRANS => $transaction_id,
             _LOCKED_FOR_TRANS_TS=>new MongoDate()
         );
-        $tripod->collection->insert($doc, array("w" => 1));
+        $collection->insert($doc, array("w" => 1));
+    }
+}
+
+class MongoTestTripod extends MongoTripod
```

Comment:
Yes, they expose properties for testing purposes that we wouldn't expose normally.